### DISCUSSION TITLE
System apps

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1264] [Help with the website]
hi i can not change system apps know in how to do

**Comment by cake03** [comment]
You cannot modify system apps in macOS 10.11 or later, but if you really want to do it then disable SIP

**Reply by tsuliwaensis** [reply]
I disabled SIP but I still couldn't change system apps. I wonder if it has something to do with the Sealed System Volume that Apple implemented with Catalina
